webhooks-extension/src/components/WebhookBranches/WebhookBranches.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from "react";
+import { Modal } from "carbon-components-react";
+import { fetchBranches } from "../../api";
+
+import {
+  DataTable,
+  DataTableSkeleton,
+  InlineNotification
+} from "carbon-components-react";
+
+const {
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableBody,
+  TableCell,
+  TableHeader
+} = DataTable;
+
+export class WebhookBranches extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rows: [],
+      loading: false,
+      error: null
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      loading: true
```

Comment:
the pattern we use elsewhere is to default to `loading: true` in the initial state